src/view/com/util/List.web.tsx

```diff
@@ -254,10 +278,12 @@ function ListImpl<ItemT>(
     },
   )
 
+  // Ignoring the error here because `nativeRef` will indeed be a `HTMLDivElement`. However, becuase we are using
+  // the RN `View`, it is not typed properly.
   return (
+    // @ts-expect-error
```

Comment:
Can we instead do something like `nativeRef as any as ...` so the rest gets checked?